modules/serverless-create-kubeconfig-file.adoc

```diff
@@ -0,0 +1,18 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="create-kubeconfig-file_{contect}"]
+= Create a `.kubeconfig` file
+
+Use kubeconfig files to organize information about clusters, users, namespaces, and authentication mechanisms. The CLI tool uses `kubeconfig` files to communicate with the API server of a cluster.
+
+Create a basic `.kubeconfig` file from client certificates.
+
+[source]
```

Comment:
Don't put source in front of this, it seems to still highlight in blue, when it shouldn't have any special highlighting at all.